cdc/owner.go

```diff
@@ -1089,6 +1139,11 @@ func (o *Owner) handleAdminJob(ctx context.Context) error {
 			if err != nil {
 				return errors.Trace(err)
 			}
+			// refuse to resume adminJob that checkPointTs lag behind gcSafePoint
+			if o.minGCSafePointCache.ts != 0 && cfInfo.StartTs < o.minGCSafePointCache.ts {
```

Comment:
refer to `GetCheckpointTs` in `model/changefeed.go` for how to get the correct checkpoint